feat(changelog): apple-silicon-changed-macos-14-is-now-available-on-all- 2024-05-17 (#3184)
scaleway/docs-content: changelog/may2024/2024-05-17-apple-silicon-changed-macos-14-is-now-available-on-all-.mdx

---
title: macOS 14 is now available on all Mac minis
status: changed
author:
    fullname: 'Join the #apple-silicon channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2024-05-17
category: bare-metal
product: apple-silicon
---

> macOS Sonoma 14 is now also installed by default on Scaleway Mac mini M1 (M1-M). Enjoy Sonoma on the entire Mac mini line!

Reviewer: ```suggestion
macOS Sonoma 14 now comes pre-installed by default on Scaleway's Mac mini M1 (M1-M). Enjoy Sonoma across the entire Mac mini lineup!
```
